### DISCUSSION TITLE
Open records by entering record type and ID into global search

### DISCUSSION THREAD

**Discussion by InitiativeWorking** [burkybang/SuiteAdvanced-Feedback#55] [Q&A] [answered]
Just an idea to add a feature so we can quickly open a record by typing [record_type]:[internalid]

It should work on the search bar and automatically replace any string texts on execution logs and text fields with a clickable link

Examples:
- project:1234
- customrecord_abc:2345
- purchaseorder:3456
- projecttask:4567

**Comment by burkybang** [comment, marked as answer]
This is already a feature!! Use a comma instead of a colon. You can even enter multiple IDs all separated by commas.

Examples:

```
project,1234
customrecord_abc, 2345
purchaseorder , 3456
projecttask   ,4567,1234, 2847
transaction, 123
entity, 123
item, 123
```

**Reply by InitiativeWorking** [reply]
oh dang I missed it! Thank you!!

**Reply by burkybang** [reply]
I'm still working on better documentation. I suspect there are plenty of unknown features yet to be discovered.

**Reply by burkybang** [reply]
@InitiativeWorking I forgot to let you know that I changed this feature to allow for using a colon (:) too!
